src/Maths/Silk.NET.Maths/Scalar.Log.cs

```diff
@@ -0,0 +1,308 @@
+﻿// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+/*
```

Comment:
Please add an accompanying comment to this second license header.

High priority.